web/src/drafts_overlay_ui.js

```diff
@@ -229,7 +229,7 @@ export function launch() {
     }
 
     const all_drafts = drafts.draft_model.get();
-    const narrow_drafts = drafts.filter_drafts_by_compose_box_and_recipient(all_drafts);
+    const narrow_drafts = drafts.filter_drafts_by_compose_box_and_recipient();
```

Comment:
This probably should pass the parameter, since we already fetched the data on the previous line and expect to use it?